spyder/app/tour.py

```diff
@@ -1388,59 +1390,114 @@ def tour_has_focus(self):
         tour_label = QLabel(
-            _("Welcome to Spyder!<br><br>Check out our interactive tour to "
+            _("Check out our interactive tour to "
               "explore some of Spyder's panes and features."))
+        tour_label.setStyleSheet(f"font-size: {self.CONTENT_FONT_SIZE}")
         tour_label.setWordWrap(True)
+        tour_label.setFixedWidth(340)
 
         # Buttons
         buttons_layout = QHBoxLayout()
         self.launch_tour_button = QPushButton(_('Start tour'))
+        self.launch_tour_button.setStyleSheet(
+          "background-color: #3775A9;"
+          f"font-size: {self.BUTTONS_FONT_SIZE};"
+          f"padding: {self.BUTTONS_PADDING}"
+        )
         self.launch_tour_button.setAutoDefault(False)
         self.dismiss_button = QPushButton(_('Dismiss'))
+        self.dismiss_button.setStyleSheet(
+          "background-color: #60798B;"
+          f"font-size: {self.BUTTONS_FONT_SIZE};"
+          f"padding: {self.BUTTONS_PADDING}"
+        )
         self.dismiss_button.setAutoDefault(False)
+
         buttons_layout.addStretch()
         buttons_layout.addWidget(self.launch_tour_button)
+        if not MAC:
+            buttons_layout.addSpacing(10)
         buttons_layout.addWidget(self.dismiss_button)
 
         layout = QHBoxLayout()
         layout.addLayout(images_layout)
-        layout.addWidget(tour_label)
-        general_layout = QVBoxLayout()
-        general_layout.addLayout(layout)
-        general_layout.addLayout(buttons_layout)
+
+        label_layout = QVBoxLayout()
+        label_layout.addWidget(tour_label_title)
+        if not MAC:
+            label_layout.addSpacing(3)
+            label_layout.addWidget(tour_label)
+        else:
+            label_layout.addWidget(tour_label)
+            label_layout.addSpacing(10)
+
+        vertical_layout = QVBoxLayout()
+        if not MAC:
+            vertical_layout.addStretch()
+            vertical_layout.addLayout(label_layout)
+            vertical_layout.addSpacing(20)
+            vertical_layout.addLayout(buttons_layout)
+            vertical_layout.addStretch()
+        else:
+            vertical_layout.addLayout(label_layout)
+            vertical_layout.addLayout(buttons_layout)
+
+        general_layout = QHBoxLayout()
+        if not MAC:
+            general_layout.addStretch()
+            general_layout.addLayout(layout)
+            general_layout.addSpacing(1)
+            general_layout.addLayout(vertical_layout)
+            general_layout.addStretch()
+        else:
+            general_layout.addLayout(layout)
+            general_layout.addLayout(vertical_layout)
+
         self.setLayout(general_layout)
 
         self.launch_tour_button.clicked.connect(self._start_tour)
         self.dismiss_button.clicked.connect(self.close)
+        self.setStyleSheet("background-color: #262E38")
+        self.setContentsMargins(18, 40, 18, 40)
+        if not MAC:
+            self.setFixedSize(640, 280)

```

Comment:
Are you sure its a good idea to rely on whatever size the OS happens to pick, which varies depending on the version, screen resolution, DPI and scaling settings? Shouldn't we set _something_ we control? Or does that mess up the layout?